vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.cpp

```diff
@@ -587,6 +592,7 @@ void ReactInstanceWin::Initialize() noexcept {
                 std::move(cxxModules),
                 m_options.TurboModuleProvider,
                 m_options.TurboModuleProvider->LongLivedObjectCollection(),
+                m_reactContext->Properties(),
```

Comment:
Worth passing the whole PropertyBag through, or should you just pass through the useRuntimeScheduler flag?

Reply:
I'm passing the PropertyBag through so that I can store the runtimeScheduler and runtimeExecutor on it before it gets to the eagerInit'd turbomodules, which will now init the FabricUIManagerBinding.

Reply:
Why do we need to init the FabricUIManagerBinding there?

Reply:
It avoids the creation of RuntimeSchedulerCallInvoker that you had.